setup.py

```diff
@@ -11,8 +11,11 @@
     long_description=long_description,
     author="Algorand",
     author_email="pypiservice@algorand.com",
-    python_requires=">=3.8",
-    install_requires=["py-algorand-sdk", "tabulate==0.8.9"],
+    python_requires=">=3.9",
+    install_requires=[
+        "py-algorand-sdk @ git+https://github.com/algorand/py-algorand-sdk@develop",
```

Comment:
Changed this to point to `develop` but still should not merge until the next release so get pull from PyPi

Reply:
`py-algorand-sdk` [v 1.1.5.0](https://pypi.org/project/py-algorand-sdk/1.15.0/) is out, so I'll revert and merge very soon.